tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = {ceph_ansible2.1}-{jewel}-{xenial_cluster,centos7_cluster}
+skipsdist = True
+
+[testenv]
+whitelist_externals =
+    vagrant
+    bash
+    pip
+    git
+
+passenv=*
+setenv=
+  TOXINIDIR = {toxinidir}
+  ANSIBLE_ACTION_PLUGINS = {toxinidir}/ceph-ansible/plugins/actions
+  # only available for ansible >= 2.2
+  ANSIBLE_STDOUT_CALLBACK = debug
+  centos7_cluster: CEPH_ANSIBLE_SCENARIO_PATH = {toxinidir}/ceph-ansible/tests/functional/centos/7/docker-cluster
+  xenial_cluster: CEPH_ANSIBLE_SCENARIO_PATH = {toxinidir}/ceph-ansible/tests/functional/ubuntu/xenial/docker-cluster
```

Comment:
There is not an ``ubuntu/xenial/docker-cluster`` testing scenario in ceph-ansible. I'm guessing we'll need to use the same scenario from ``centos/7/docker-cluster`` but tell it to use a different image.